pyrightconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "include": ["examples/**/*.py", "test/**/*.py"],
+    "exclude":["venv", "*.pyc", "*.egg-info", ".coverage"],
+
```

Comment:
Empty line for no reason?

Reply:
This is to separate out the include, exclude keys with the rest of the checks.

Reply:
No need to do that.